README.md

```diff
@@ -76,9 +76,9 @@ The `spec` configures how the index should be deployed. For serverless indexes,
 
 ```typescript
 import { Pinecone } from '@pinecone-database/pinecone';
-const pc = new Pinecone();
```

Comment:
I can swap back to `pc` that's fine. If we want to standardize I think there are a few Python docstrings that are using `client` atm which was why I swapped to this. 😅 

https://github.com/pinecone-io/pinecone-python-client/blob/8f2c370245dbba615c9a88440bd19f1e522692a4/pinecone/control/pinecone.py#L145

Reply:
Good callout on the python example inconsistency. I do prefer `pc` I think because "client" is a general concept. For example you get an index object, that is also a client.